dora/tests/integration/src/test/java/alluxio/client/cli/fsadmin/command/ReportEmbeddedJournalIntegrationTest.java

```diff
@@ -1,71 +0,0 @@
-/*
- * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
- * (the "License"). You may not use this work except in compliance with the License, which is
- * available at www.apache.org/licenses/LICENSE-2.0
- *
- * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
- * either express or implied, as more fully set forth in the License.
- *
- * See the NOTICE file distributed with this work for information regarding copyright ownership.
- */
-
-package alluxio.client.cli.fsadmin.command;
-
-import alluxio.ProjectConstants;
-import alluxio.annotation.dora.DoraTestTodoItem;
-import alluxio.client.cli.fsadmin.AbstractFsAdminShellTest;
-import alluxio.conf.Configuration;
-import alluxio.conf.PropertyKey;
-import alluxio.testutils.LocalAlluxioClusterResource;
-import alluxio.util.network.NetworkAddressUtils;
-
-import org.hamcrest.CoreMatchers;
-import org.junit.Assert;
-import org.junit.Ignore;
-import org.junit.Test;
-
-@LocalAlluxioClusterResource.ServerConfig(
-    confParams = {
-        PropertyKey.Name.MASTER_JOURNAL_TYPE, "EMBEDDED",
-        PropertyKey.Name.MASTER_EMBEDDED_JOURNAL_PORT,
-        ReportEmbeddedJournalIntegrationTest.JOURNAL_PORT})
-@Ignore
-@DoraTestTodoItem(action = DoraTestTodoItem.Action.REMOVE, owner = "Jiacheng",
```

Comment:
the journal feature no longer matters in 3.x. The master will no longer have a journal. The journal code still exits as of now but will be removed soon.